### ISSUE TITLE
Variadic functions in TypeScript

### ISSUE THREAD

**Issue by sqs** [sourcegraph/issues#144]

**Comment by felixfbecker** [comment]
I don't think this is related to variadic functions, and more to the fact that the TypeScript lib.d.ts files don't have a tsconfig file (i.e. TypeScript-repo-specific setup).

**Comment by sqs** [comment]
Cool. Please triage. I reported it from my phone from a user report.
